src/modules/MainConnector.js

```diff
@@ -17,24 +14,17 @@ import {enableScan, disableScan} from './UI/scenes/Scan/action'
 
 import makeContextCallbacks from './Core/Context/callbacks'
 
-const mapStateToProps = (state: State) => ({
-  routes: state.routes,
-  autoLogoutTimeInSeconds: SETTINGS_SELECTORS.getAutoLogoutTimeInSeconds(state)
-})
+const mapStateToProps = () => ({})
+
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  dispatchEnableScan: () => {
-    return dispatch(enableScan())
-  },
-  dispatchDisableScan: () => {
-    return dispatch(disableScan())
-  },
+  dispatchEnableScan: () => dispatch(enableScan()),
```

Comment:
Please don't collapse these constructs if they are already coded as such. Collapsing them removes the ability to set a breakpoint. Please revert these lines